recipes/ssh2-python/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ssh2-python" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "e5e77801a7a0d49799e288ecbd189b4d1c23eeadcf9c6f4ce80b31b7d397babe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
```

Comment:
This shouldn't be needed for a Python library. So can be dropped.

Reply:
Removed, thanks for reference. I read somewhere they were needed as it's technically allowed for extensions to be built with a different compiler than the interpreter.

Reply:
Basically we have decided not to support non-standard CPython/VC combinations in conda-forge Where did you read this as it sounds like it needs fixing?